bundles/org.openhab.binding.panamaxfurman/src/main/java/org/openhab/binding/panamaxfurman/internal/protocol/PowerConditionerChannel.java

```diff
@@ -0,0 +1,119 @@
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.panamaxfurman.internal.protocol;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.regex.Matcher;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.panamaxfurman.internal.PanamaxFurmanAbstractHandler;
+import org.openhab.binding.panamaxfurman.internal.PanamaxFurmanConstants;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.types.State;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Enum for channel that is exposed by the device
+ *
+ * @author Dave Badia - Initial contribution
+ */
+@NonNullByDefault
+public enum PowerConditionerChannel {
+    POWER("power", OnOffType.class, true),
+    ID("id", null, false),;
+
+    private final static Logger logger = LoggerFactory.getLogger(PowerConditionerChannel.class);
+
+    private final static Map<String, PowerConditionerChannel> CHANNEL_STRING_TO_ENUM_TABLE = new ConcurrentHashMap<>();
+
+    private final String name;
+    private final @Nullable Class<? extends State> stateClass;
+    private final boolean outletSpecific;
+
+    private PowerConditionerChannel(String name, @Nullable Class<? extends State> stateClass, boolean outletSpecific) {
+        this.name = name;
+        this.stateClass = stateClass;
+        this.outletSpecific = outletSpecific;
+    }
+
+    public String getChannelName(@Nullable Integer outletNumber) {
+        if (outletNumber == null) {
+            return getName();
+        }
+        if (isOutletSpecific()) {
+            return PanamaxFurmanAbstractHandler.getChannelUID(name, outletNumber);
+        } else {
+            logger.warn("{} was called with outlet #{} but is not outlet specific. ()", this, outletNumber);
+            return name;
+        }
+    }
+
+    public String getName() {
+        if (isOutletSpecific()) {
+            throw new IllegalStateException(this.name() + " was called without an outlet # but is outlet specific");
+        }
+        return name;
+    }
+
+    public @Nullable Class<? extends State> getStateClass() {
+        return stateClass;
+    }
+
+    public @Nullable State buildState(String stateValueParam) {
+        String stateValue = stateValueParam.trim();
+        if (getStateClass() == null) {
+            return null;
+        } else if (OnOffType.class.equals(getStateClass())) {
+            return OnOffType.from(stateValue);
+        } else {
+            logger.warn("Don't know how to map state class {}, returning null", stateClass);
```

Comment:
```suggestion
            logger.warn("Don't know how to map state class {}, returning null", stateClass);
```

There are more warn level, are you sure all these needs to be warn instead of debug ? Is there anything the user can do about it or should act (like raise an issue for unsupported device or something liek that)

Reply:
Yes, the user should definitely raise an issue if this particular warning occurs.  I'll double check the other warn to see if I can reduce the log level

Reply:
Her eyou could also help the user to tell him what to do:
"Can't map state class {}, returning null. Please raise a Github issue"

Reply:
Good point, updated